javascript/javascript1/week4/voiceAssistantTask.js

```diff
@@ -0,0 +1,119 @@
+// Voice Assistant Application:
+
+let personName = "";
+const toDos = [];
+
+//Some of the motivational quotes:
+const quotes = [
+  "Keep your face always toward the sunshine, and shadows will fall behind you.",
+  "Everything you've ever wanted is sitting on the other side of fear.",
+  "Attitude is a little thing that makes a big difference.",
+  "I am not a product of my circumstances.",
+  "If you can dream it, you can do it.",
+];
+
+// For nice to meet you, benjamin.
+function getReply(command) {
+  command = command.toLowerCase();
```

Comment:
Excellent idea of using `toLowerCase()`! You can also use `.trim()` to clean the user input even more.

Reply:
.trim() function implemented. Thank you for suggestion!